docs/MigrationGuide.md

```diff
@@ -519,3 +521,10 @@ The following addons have been removed/merged with other addons:
 - The `ecs_term_copy`/`ecs_term_move`/`ecs_term_fini` functions have been removed. Terms no longer exist by themselves, and term resources are now owned by the query object.
 - The `ecs_iter_poly` function has been removed. To iterate all entities in the world, now use `ecs_get_entities`.
 - `ecs_field_column` has been renamed to `ecs_field_column_index`.
+- `FLECS_TERM_DESC_MAX` has been renamed to `FLECS_TERM_COUNT_MAX`.
+- `ecs_app_desc_t::enable_monitor` has been renamed to `ecs_app_desc_t::enable_stats`.
+- `ecs_singleton_ensure` has been renamed to `ecs_singleton_add`.
```

Comment:
Remove this line. It is not correct , `ecs_singleton_ensure` still exists.

Reply:
Oops, I should have missed something while fixing my code